### PR TITLE
Change ansible documentation

### PR DESCRIPTION
**Pull request by Settis**
Add description about openstack ansible upgrade.

### CODE REVIEW

File: doc/source/ansible_driver.rst [hunk 1]
```diff
@@ -63,6 +63,9 @@ Here is an example of terminating:
     p.join()
 
 
+In case when we have multiple servers flow will be the following. Ansible iterates by tasks and for each task iterates by servers.
```
Comment (sc68cal) [inline doc/source/ansible_driver.rst]: break up into two lines

File: doc/source/ansible_driver.rst [hunk 2]
```diff
@@ -145,4 +148,18 @@ Conclusion
...
+For running upgrade you must execute `scripts/run-upgrade.sh`.
+
+This script consists of two parts:
+
+* run `scripts/bootstrap-ansible.sh` for install required tools, packages, etc.
+* run sequence of 20 ansible playbooks
+
+Each playbook runs independently, they don't have shared namespace with variables.
+
+That means that we can split whole upgrade at least on 20 peaces. Also if the peace was done successfully we can not repeat it when try to resume upgrade.
```
Comment (sc68cal) [inline doc/source/ansible_driver.rst]: pieces

File: doc/source/ansible_driver.rst (hunk 1 above)
Comment (Settis) [inline doc/source/ansible_driver.rst]: done

File: doc/source/ansible_driver.rst (hunk 2 above)
Comment (Settis) [inline doc/source/ansible_driver.rst]: Ooops! :) Done.

**Pull request merged**